### PR TITLE
Fix link to PDF docs

### LINKED ISSUE #1917: Link to PDF documentation is broken
The main pytest page (http://docs.pytest.org/en/latest/) links to PDF documentation in the sidebar:

http://docs.pytest.org/en/latest/pytest.pdf

Visiting that link takes me to a 404 page on ReadTheDocs.

### PR DESCRIPTION
**Pull request by The-Compiler**
Fixes #1917

### CODE REVIEW

File: doc/en/_templates/links.html [hunk 1]
```diff
@@ -6,6 +6,6 @@
   <li><a href="https://github.com/pytest-dev/pytest/">pytest @ GitHub</a></li>
   <li><a href="http://plugincompat.herokuapp.com/">3rd party plugins</a></li>
   <li><a href="https://github.com/pytest-dev/pytest/issues">Issue Tracker</a></li>
-  <li><a href="http://pytest.org/latest/pytest.pdf">PDF Documentation</a>
```
Comment (RonnyPfannschmidt) [inline doc/en/_templates/links.html]: perhaps the link should show to the exact own release on rtd
else older releases will point to the up2date pdf

Comment (The-Compiler) [inline doc/en/_templates/links.html]: I don't see how that'd be possible without us adjusting three PDF links with every release.

Comment (nicoddemus) [inline doc/en/_templates/links.html]: @RonnyPfannschmidt I agree that would be ideal, but as @The-Compiler said I also don't see how to do that in any automated manner.

FWIW this link was always pointing to the "latest" before moving to RTD, so unless we have a suggestion on how to easily update this link automatically I would rather have the broken link fixed now than later. :grin:

**Comment by RonnyPfannschmidt** [comment]
Let's make an issue and ask rtd?

**Comment by nicoddemus** [comment]
Sure sure, if they have a solution it would be most welcome! :grin:

**Comment by The-Compiler** [comment]
So can we merge this, since that is unrelated to this PR?

**Comment by nicoddemus** [comment]
Thanks @The-Compiler!

**Pull request merged**